From 95d8f16ece96829109eb75a605b1459a25b5a6be Mon Sep 17 00:00:00 2001
From: Ondrej Mosnacek <omosnace@redhat.com>
Date: Thu, 23 Jan 2020 13:57:15 +0100
Subject: [PATCH] checkpolicy: remove unused te_assertions

This variable is declared in a header file, but never defined or used.
The te_assert structure definition is only used in this declaration, so
remove both.

Signed-off-by: Ondrej Mosnacek <omosnace@redhat.com>
---
 checkpolicy.h | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/checkpolicy.h b/checkpolicy.h
index 3868f1fa023b..f127687e72a6 100644
--- a/checkpolicy.h
+++ b/checkpolicy.h
@@ -1,20 +1,6 @@
 #ifndef _CHECKPOLICY_H_
 #define _CHECKPOLICY_H_
 
-#include <sepol/policydb/ebitmap.h>
-
-typedef struct te_assert {
-	ebitmap_t stypes;
-	ebitmap_t ttypes;
-	ebitmap_t tclasses;
-	int self;
-	sepol_access_vector_t *avp;
-	unsigned long line;
-	struct te_assert *next;
-} te_assert_t;
-
-te_assert_t *te_assertions;
-
 extern unsigned int policyvers;
 
 #endif
-- 
2.24.1

